libtelemetry/src/test/java/com/mapbox/android/telemetry/LocationMapperTest.java

```diff
@@ -1,16 +1,28 @@
 package com.mapbox.android.telemetry;
 
+import android.app.ActivityManager;
+import android.content.Context;
 import android.location.Location;
 
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class LocationMapperTest {
 
+  @Before
```

Comment:
In general, we should avoid inline setup 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=65
Could you extract `test` code into a `private` "factory" method called for example `obtainMockedContext` and use it only in the tests that actually need it?
IMO, it reflects better what's going on in the test and you don't break the [triple A](https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40) "rule".